example/l2fwd_simple/odp_l2fwd_simple.c

```diff
@@ -106,8 +107,11 @@ static int run_worker(void *arg ODP_UNUSED)
 		pkts = odp_pktin_recv_tmo(global.if0in, pkt_tbl, MAX_PKT_BURST,
 					  wait_time);
 
-		if (odp_unlikely(pkts <= 0))
+		if (odp_unlikely(pkts <= 0)) {
+			if (exit_10pkts)
```

Comment:
`if (exit_10pkts--)`
This could be a count to mean how many seconds of silence in pktin we can accept before exiting the loop (see below)

Reply:
uh, yes,  total packets is bellow. This variable is just break or not break this loop. I will send update.